dissect/target/tools/diff.py

```diff
@@ -235,14 +235,23 @@ def scandir(self, path: str) -> DirectoryDifferential:
             src_fh = src_entry.open()
             dst_fh = dst_entry.open()
 
+            # We reverse read the file object.
+            chunk_size = 1024 * 10
+            chunks_a = fsutil.reverse_read(src_fh, chunk_size, reverse_chunk=False)
+            chunks_b = fsutil.reverse_read(dst_fh, chunk_size, reverse_chunk=False)
+
+            chunk_count = 0
+
             while True:
-                chunk_a = src_fh.read(BLOCK_SIZE)
```

Comment:
Is `BLOCK_SIZE` still used anywhere? Otherwise it could be removed, or re-used as the chunk size.